### DISCUSSION TITLE
How to prevent open 404.page on .well-known/apple-app-site-association

### DISCUSSION THREAD

**Discussion by kanopa** [nuxt/nuxt#17400] [Questions]
I store .well-known/apple-app-site-association on public directory. On try to open, file not opened and redirect to 404 page 

[Vue Router warn]: No match found for location with path "/.well-known/apple-app-site-association"
[Vue Router warn]: No match found for location with path "/.well-known"

**Comment by kissu** [comment]
Similar question asked here: https://stackoverflow.com/questions/72712551/nuxt-3-well-known-route
